### DISCUSSION TITLE
Vercel Serverless Python Deployment Randomly Went Offline - Still works locally

### DISCUSSION THREAD

**Discussion by neelS-hah** [vercel/community#4362] [Help]
### Summary

Python runtime serverless deployment was working up until last night - none of the previous deployment links are working anymore. The function still works locally - even moving it to a new vercel project didn't get it to work.

This is specifically for and image to pdf conversion - 
from http.server import BaseHTTPRequestHandler, HTTPServer
import cgi
import json
import base64
from pdf2image import convert_from_path
from io import BytesIO
import fitz 

these are the packages it's using

### Steps to Reproduce

logs from pdf2jpeg in my account, for POST request

**Comment by amyegan** [comment]
If there's any related [build](https://vercel.com/docs/deployments/logs#) or [runtime log](https://vercel.com/docs/observability/runtime-logs#) info, that would be good to share. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that folks can use to help with debugging?
